Index: src/main/java/dev/gutierrez/utils/ConnectionUtil.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package dev.gutierrez.utils;public class ConnectionUtil {\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/dev/gutierrez/utils/ConnectionUtil.java b/src/main/java/dev/gutierrez/utils/ConnectionUtil.java
--- a/src/main/java/dev/gutierrez/utils/ConnectionUtil.java	(revision 17dae00b717e8946067edf02077de951e7ebf9b3)
+++ b/src/main/java/dev/gutierrez/utils/ConnectionUtil.java	(date 1659553045162)
@@ -1,2 +1,4 @@
-package dev.gutierrez.utils;public class ConnectionUtil {
+package dev.gutierrez.utils;
+
+public class ConnectionUtil {
 }
